### ISSUE TITLE
Git: title actions are missing name and role

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52278]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10 / HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038 , VoiceOver

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Source Control"(Cntrl+Shift+G) button.
3. Navigate to treeview below edit field and within each item use "Tab" key to navigate to "Stage/Unstage all changes","Stage/Unstage changes","Open File" etc.

**Actual:**
Screenreader does not read the name,role,value and other information for controls "Stage/Unstage all changes","Stage/Unstage changes","Open File" etc. represnted using icons for the same.

**Same issue has been observed with JAWS and VoiceOver**

**Expected:**
Screenreader should read the name,role,value and other information for controls "Stage/Unstage all changes","Stage/Unstage changes","Open File" etc.

**Recommendations:**
Add proper name and role to the Stage/Unstage all changes","Stage/Unstage changes","Open File" etc. controls.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the name,role,value,state or any other information related to the controls present on the screen.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_SourceControl_ScreenReader_ControlIssue.pptx](https://github.com/Microsoft/vscode/files/2114297/A11y_VSCode_SourceControl_ScreenReader_ControlIssue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by joaomoreno** [comment]
Dupe of #52682

**Comment by mjbvz** [comment]
@joaomoreno Is this a duplicate or was something fixed for this iteration?

**Comment by chrmarti** [comment]
Not fixed, updating labels.
